IdentityCore/src/cache/accessor/MSIDAccountCredentialCache.m

```diff
@@ -94,11 +94,13 @@ - (instancetype)initWithDataSource:(id<MSIDExtendedTokenCacheDataSource>)dataSou
 
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->